app/api/stripe/webhook/route.ts

```
@@ -1,197 +1,77 @@
+      console.error('Webhook signature verification failed:', err)
+      return NextResponse.json(
+        { success: false, error: 'Invalid signature' },
+        { status: 400 }
+      )
+    }
 
+    // Handle the event
     switch (event.type) {
-      case "checkout.session.completed": {
+      case 'checkout.session.completed':
         const session = event.data.object as Stripe.Checkout.Session
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)